imports/ui/explorer-components/filter-row.jsx

```diff
@@ -1,38 +1,118 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
+import {
+  selectInputIconStyle,
+  noUnderline,
+  sortBoxInputStyle,
+  selectedItemStyle,
+  sortBoxBlueInputStyle
+} from '../components/form-controls.mui-styles'
+import MenuItem from 'material-ui/MenuItem'
+import SelectField from 'material-ui/SelectField'
+import { labels } from '../explorer-components/sort-items'
 
 export class FilterRow extends Component {
```

Comment:
It looks like this component is now a mashup of both the filter and the sort. If it's impossible to make this component agnostic to whether it's a filter or a sort and work in a more coherent way, then it should be split into two separate components with separate functionallities